### ISSUE TITLE
Rename run options

### ISSUE THREAD

**Issue by MaximMoinat** [thehyve/delphyne#166]
**How would this feature be useful?**
The current run options with `skip_` cause a double negative which is very confusing.
(and yes, I was myself guilty of introducing the `skip_` syntax in the first place...)

**Describe the solution you'd like**
Rename
```yaml
run_options:
  skip_vocabulary_loading: False
  skip_custom_vocabulary_loading: False
  skip_source_to_concept_map_loading: False
```
to
```yaml
run_options:
  do_load_standard_vocabulary: True
  do_load_custom_vocabulary: True
  do_load_source_to_concept_map: True
```

**Comment by alepev** [comment]
Completely agree with this

**Comment by Spayralbe** [comment]
Agreed. Let's change this while we can still sort of get away with making breaking changes :p

**Comment by MaximMoinat** [comment]
On that; how are we going to track breaking changes between releases? (as in changes that need to be made to the ETL when going from one version to another)

I assume either in the release notes or in the documentation. (or both?)

**Comment by Spayralbe** [comment]
Preferably we don't introduce any breaking changes within the same major version.
When breaking changes are made, I think they should be listed in the changelog (docs will already be version specific).
Now we're still in a pre-release stage though, so only after we have released 0.1.0 should we start carefully tracking this in the changelog.

Of course you could always pinpoint your dependencies in an ETL repo to an exact version of delphyne, such that your pipeline won't be affected if a new release introduces breaking changes.
